tests/test_importexport.py

```diff
@@ -2265,6 +2265,20 @@ def test_bench_with_same_io_name(self):
         self.check_io(pyrtl.Input, ["G1", "G2", "G3"])
         self.check_io(pyrtl.Output, ["tmp3", "G4"])
 
+class TestOutputVerilog(unittest.TestCase):
+    def test_custom_module_name(self):
```

Comment:
I think it makes more sense to add this new test to the existing `TestVerilogOutput` defined on L1139, rather than defining a new `TestOutputVerilog`, so we can keep all the `output_to_verilog` tests together.